lib/exports/mixins.less

```diff
@@ -110,6 +110,40 @@
 }
 
 
+/**
+ * Focus styles for the given context.
+ *
+ * Usage example:
+ * .focus-styles(true, true);
+ *
+ * @inset: boolean - whether the focus style be placed inside the element.
+ * @border: boolean - whether the element's border color change to match the focus style.
+ */
+.focus-styles(@inset: false, @border: false) {
+    & when (@inset = false) {
```

Comment:
No action required here but I wanted to mention that personally I find easier to read when there are less level of nesting. For example:
```less
    & when not (@inset) and not (@border) {
        box-shadow: 0 0 0 var(--su-static2) var(--focus-neutral), 0 0 0 var(--su-static4) var(--focus-theme);
    }

    & when not (@inset) and (@border) {
        border-color: var(--focus-neutral) !important;
        box-shadow: 0 0 0 var(--su-static1) var(--focus-neutral), 0 0 0 calc(var(--su-static4) - var(--su-static1)) var(--focus-theme);
    }

    & when (@inset) and not (@border) {
        box-shadow: inset 0 0 0 var(--su-static2) var(--focus-theme), inset 0 0 0 var(--su-static4) var(--focus-neutral);
    }

    & when (@inset) and (@border) {
        border-color: var(--focus-theme) !important;
        box-shadow: inset 0 0 0 var(--su-static1) var(--focus-theme), inset 0 0 0 calc(var(--su-static4) - var(--su-static1)) var(--focus-neutral);
    }
```
Wanted to double check what's your preference. I am happy to leave things as they are. 🙂

Reply:
Good call! I've made this change in https://github.com/StackExchange/Stacks/pull/1629/commits/1f6e0fa717b82a2f294df2e88f2fe33e5bdcea95